offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/create/Organisation.groovy

```diff
@@ -0,0 +1,44 @@
+package life.qbic.portal.offermanager.components.person.create
+
+import life.qbic.datamodel.dtos.business.Affiliation
+
+/**
+ * <h1>Organisation of an affiliation</h1>
+ * <br>
+ * <p>An address organisation can have multiple affiliations. In order to correctly map between an organisation and
+ * all of its affiliations</p>
+ *
+ * @since 1.0.0
+ */
+class Organisation {
+
+    /**
+     * The name of the organisation
+     * @since 1.0.0
+     */
+    final String name
+    /**
+     * The affiliation associated to that organisation.
+     * It needs to be immutable so that new affiliations can be added
+     *
+     * @since 1.0.0
+     */
+    List<Affiliation> affiliations
+
+    Organisation(Affiliation affiliation) {
+        this.name = affiliation.organisation
+        this.affiliations = new ArrayList<Affiliation>([affiliation])
+    }
+
+    /**
+     * Creates an organisation with an organisation name and all of its affiliations
+     * @param name
+     * @param affiliations
```

Comment:
Maybe NIT: I think we normally specify the parameters even if it's redundant and we have a space between method description and parameter listing 
```suggestion
     * Creates an organisation with an organisation name and all of its affiliations
     *
     * @param name name of the organisation
     * @param affiliations list of affiliations associated with the organisation
```